feature_extraction:
    ros__parameters:
        cloud_msg_cache_size: 2
        max_lidar_range: 30.0
        min_box_x: -0.3
        max_box_x: 0.8
        min_box_y: -0.25
        max_box_y: 0.25
        min_box_z: -0.3
        max_box_z: 0.3
        threshold_edge: 1.0
        threshold_flat: 0.1
        flat_leaf_size: 0.1
graph_optimization:
    ros__parameters:
        flat_leaf_size: 0.2
        edge_leaf_size: 0.05
        ICP_leaf_size: 0.2
        local_map_size: 100
        rad_new_key_frame: 0.17
        dist_new_key_frame: 0.2
        loop_search_dist: 2.0
        loop_closure_frequency: 1.0
